src/deye_daemon.py

```diff
@@ -109,6 +117,36 @@ def __remove_duplicated_reg_ranges(self, reg_ranges: list[SensorRegisterRange])
                 result.append(reg_range)
         return result
 
+    def __new_event(self, events: list[DeyeEvent]) -> bool:
```

Comment:
Please add unit tests for this method.

Reply:
added unit tests for this method of DeyeDaemon